datadog/dogshell/common.py

```diff
@@ -77,6 +77,8 @@ def load(self, config_file, api_key, app_key):
 
             self['api_key'] = config.get('Connection', 'apikey')
             self['app_key'] = config.get('Connection', 'appkey')
+            if config.has_section('Proxy'):
+                self['proxies'] = config._sections['Proxy']
```

Comment:
```suggestion
                self['proxies'] = dict(config.items('Proxy'))
```
Let's do like this to not use the private attribute directly